docs/guides/capability-sets.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Session Capability Sets For Common Testing Scenarious
+---
+
+This article describes necessary capabilities that must be provided in order
+to implement some common automation testing scenarios.
+It only describes very minimum sets of capabilities required to
+be included. For refined setups more of them might need to be provided. Check the
+[Capabilities](../reference/capabilities.md) article for more details
+on each option available for the fine-tuning of XCUITest driver sessions.
+
+### Simple iOS App Test On a Real Device
+
+```json
+{
+  "platformName": "mac",
+  "appium:automaitionName": "XCUITest",
+  "appium:platformVersion": "<iOS_Version>",
```

Comment:
Personally I don't really provide `platformVersion` for real devices, since the driver can autodetect it anyway. But I guess it's just good practice?